lib/mozart_fetcher/http_client.ex

```diff
@@ -3,10 +3,9 @@ defmodule HTTPClient do
 
   alias MozartFetcher.TimeoutParser
 
-  def get(endpoint, client \\ client()) do
+  def get(endpoint, headers \\ [], client \\ client()) do
```

Comment:
Probably we can add the headers after the client and then will not need to change some other tests or having it called somewhere with second option client. What do you think?

Reply:
The client is only used for mocking in tests. I would prefer to keep the usable parameters together and ideally first.